src/components/item/settings/ThumbnailSetting.tsx

```diff
@@ -115,14 +116,14 @@ const ThumbnailSetting = ({ item }: Props): JSX.Element | null => {
   };
 
   const alt = translateBuilder(BUILDER.THUMBNAIL_SETTING_MY_THUMBNAIL_ALT);
-  const defaultImageComponent = <img src={defaultImage} alt={alt} />;
+  const imgUrl = thumbnailUrl ?? defaultImage;
 
   return (
     <>
       {uppy && (
         <StatusBar uppy={uppy} handleClose={handleClose} open={openStatusBar} />
       )}
-      <Grid container justifyContent="space-between">
+      <Stack spacing={3} mb={3}>
         <Grid item sm={6}>
```

Comment:
Inside a `Stack` you should not need to use the `<Grid item>` component. You can simply have the different elements as direct children no wrapper needed.